home/home5.2/src/ConfigLoader.java

```diff
@@ -0,0 +1,43 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ConfigLoader {
+    private static ConfigLoader instance;
+    private final Map<Character, String[]> config = new HashMap<>();
+
+    private ConfigLoader() {
+        loadConfig();
+    }
+
+    public static synchronized ConfigLoader getInstance() {
+        if (instance == null) {
+            instance = new ConfigLoader();
+        }
+        return instance;
+    }
+
+    private void loadConfig() {
+        try (BufferedReader reader = new BufferedReader(new FileReader("config.txt"))) {
```

Comment:
config.txt should be a named constant on the higher level of the program(the code that will use this class) and here we can  pass it as an input argument. Such an approach gives us more flexibility in terms of testing and support of the program